graylog_hook.go

```diff
@@ -40,6 +41,12 @@ type graylogEntry struct {
 	line int
 }
 
+func NewGraylogLevelHook(addr string, extra map[string]interface{}, levels []logrus.Level) *GraylogHook {
+	hook := NewGraylogHook(addr, extra)
+	hook.levels = levels
```

Comment:
I'd do it how `logrus.SetLevel` does it.

```golang
  // Only log the X severity or above.
  log.SetLevel(log.XLevel)
```

`NewGraylogLevelHook(addr, extra, minLevel logrus.Level)`